tracer/test/snapshots/AspNetCoreIisMvc30Tests.OutOfProcess.WithFF.__path=_not-found_statusCode=404.verified.txt

```diff
@@ -9,11 +9,14 @@
     Tags: {
       component: aspnet_core,
       env: integration_tests,
+      http.client_ip: 127.0.0.1,
```

Comment:
No `http.route` when it's a 404, which makes sense, but is that the correct behaviour?

Reply:
same as other, as it's optional and a best effort needed.. I guess it's ok for now